cmd/snap/cmd_snap_op.go

```diff
@@ -142,7 +146,19 @@ func (x *cmdInstall) Execute([]string) error {
 		return err
 	}
 
-	return wait(cli, changeID)
+	if err := wait(cli, changeID); err != nil {
+		return err
+	}
+
+	// extract the snapName from the change, important for sideloaded
```

Comment:
we know from above if it's a sideload or not, feels we should error more if it's a sideload and we don't get actually get a name